File: 1 - Two Sum/main.cpp
#pragma once
#include <vector>

using namespace std;

vector<int> twoSum(vector<int>& nums, int target);
vector<int> twoSum(vector<int>& nums, int target) {
        
    vector<int> indices(2,0);
    int vector_size = nums.size();
        
        
    for(int i = 0 ; i < vector_size; i++)
    {
        for(int j = 0; j < vector_size; j++)
        {
            if(i == j)
            {
                continue;
            }
            if(nums[i] + nums[j] ==  target)
            {
                indices[0] = i;
                indices[1] = j;
                    
                return indices;
            }
        }
    }
    return indices;
}

File: 1 - Two Sum/main.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        Dictionary<int, int> numbers = new Dictionary<int, int>();

        for(int i = 0; i < nums.Length; i++){
            if(numbers.ContainsKey(target - nums[i])){
                return [numbers[target - nums[i]], i];
            }
            numbers[nums[i]] = i;
        }

        return [-1,-1];
    }
}